benchmarks/dynamo/ci_expected_accuracy/cpu_inductor_huggingface_freezing_inference.csv

```diff
@@ -0,0 +1,185 @@
+name,accuracy,graph_breaks
+
+
```

Comment:
Why there are some many space line?

Reply:
I copied the template from cpu_inductor_huggingface_inference.csv. The format is supposed to align with it.